third_party/CMakeLists.txt

```diff
@@ -101,6 +101,30 @@ if(${NUMA_FOUND})
     add_subdirectory(pgasus)
 endif()
 
+## Build lz4
+set(LZ4_LIBRARY_DIR lz4/lib)
+
+add_library (lz4
+    ${LZ4_LIBRARY_DIR}/lz4.c
+    ${LZ4_LIBRARY_DIR}/lz4hc.c
+
+    ${LZ4_LIBRARY_DIR}/lz4.h
+    ${LZ4_LIBRARY_DIR}/lz4hc.h
+)
+
+target_compile_definitions(
+    lz4
+
+    PUBLIC
+    LZ4_DISABLE_DEPRECATE_WARNINGS=1)
```

Comment:
Is this a workaround for an lz4 bug or are you actually using deprecated lz4 functions? If so, why?

Reply:
No that's just a copy and paste remnant from when we fixed our lz4 integration (i.e., compiling and linking). We'll remove it.